src/components/StaleDataIndicator.test.js

```diff
@@ -17,17 +17,26 @@ it("shows no indicator if the data is less than 10 minutes old", async () => {
 
 describe("staleness messaging", () => {
     it("shows time if it's today", async () => {
-        const minutesStaleTimestamp = new Date().setHours(12, 0, 0, 0); // 35 mins stale
+        // Use the smallest possible difference in time, to avoid failing tests
+        // when run near the boundary of two days in real time.
+        const minutesStaleTimestamp = new Date() - 1; // 1 millisecond old
         await act(async () => {
             render(
                 <StaleDataIndicator
                     timestamp={minutesStaleTimestamp}
-                    staleMinutesOverride={10}
+                    staleMinutesOverride={0}
                 />
             );
         });
 
-        expect(screen.getByText("Last updated 12:00 PM")).toBeTruthy();
+        const timestampDate = new Date(minutesStaleTimestamp);
+        expect(
+            screen.getByText(
+                `Last updated ${
+                    timestampDate.getHours() % 12
+                }:${timestampDate.getMinutes()} PM`
```

Comment:
won't this fail in the morning now? Since it says PM?

Reply:
Yep! I'll have to add a ternary here based on whether hours > 12.